### PR TITLE
Add ukrainian-only messagease symbols layout

### PR DESCRIPTION
**Pull request by Meetoul**

### CODE REVIEW

File: ThumbKey.apk
Comment (dessalines) [inline ThumbKey.apk:1]: Needs to be removed.

**Review by dessalines — CHANGES_REQUESTED** [review]

File: ThumbKey.apk
Comment (Meetoul) [reply ThumbKey.apk:1]: Ahh, sorry. Removed!

Comment (dessalines) [reply ThumbKey.apk:1]: No probs!

**Review by dessalines — APPROVED** [review]
Thx! This will be in the next release.

**Pull request merged**